docs/reference/indices/delete-data-stream.asciidoc

```diff
@@ -0,0 +1,29 @@
+[[indices-delete-data-stream]]
+=== Delete data stream API
+++++
+<titleabbrev>Delete data stream</titleabbrev>
+++++
+
+Deletes an existing data stream along with its backing indices.
+
+[source,console]
+--------------------------------------------------
+DELETE _data_stream/my-data-stream
+--------------------------------------------------
+// TEST[setup:twitter]
+
+
+[[delete-data-stream-api-request]]
+==== {api-request-title}
+
+`DELETE _data_stream/<data-stream>`
+
+
+[[delete-data-stream-api-path-params]]
+==== {api-path-parms-title}
+
+`<data-stream>`::
++
+--
+(Required, string) Name or wildcard expression of data stream(s) to delete.
```

Comment:
This is created an unterminated block, which is breaking the docs build. Since the definition is a single para, we don't need the `+` or `--` (block).
```suggestion
`<data-stream>`::
(Required, string) Name or wildcard expression of data stream(s) to delete.
```